lib/libspl/timestamp.c

```diff
@@ -28,6 +28,10 @@
 #include <langinfo.h>
 #include "statcommon.h"
 
+#ifndef _DATE_FMT
```

Comment:
It would probably be nicer to move this into our own langinfo.h header that uses `#include_next <langinfo.h>`. The new file should be under CDDL 1.0 only, rather than unbound to be consistent with how all new files are added.